x-pack/test/security_solution_cypress/es_archives/unmapped_fields/mappings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "index",
+  "value": {
+    "aliases": {
+      "mydata-7.12.0": {
+        "is_write_index": true
+      }
+    },
+    "index": "unmapped-7.12.0-2021.03.10-000001",
+    "mappings": {
+      "dynamic": false,
+      "properties":{
+        "@timestamp":{
+          "type":"date"
+        },
+        "mydestination":{
+          "properties":{
+            "ip":{
+              "index": "false",
+              "type":"keyword"
+            }
+          }
+        }
+      }
+    }
+    }
+}
```

Comment:
A few other settings I would recommend that a lot of devs are using within their integration tests:

```ts
        "number_of_replicas": "1",
        "number_of_shards": "1",
        "refresh_interval": "5s"
```

Example you did earlier using them:
https://github.com/elastic/kibana/blob/master/x-pack/test/security_solution_cypress/es_archives/threat_indicator/mappings.json#L818

I don't know if es-archiver uses some of these defaults or can such as `refresh_interval: "5s"` but the refresh_interval of 5s is pretty good as it makes a lot of inconsistency data issues with loading/unloading better